tada/util/generate.py

```diff
@@ -19,13 +20,22 @@ def generate_data(chosen_types, chosen_size):
         generated_values = generated_values + (generated_value,)
     return generated_values
 
-
+# pylint: disable=unused-argument
 def generate_int(chosen_size):
     """Generate an int value"""
-    return int(chosen_size)
+    # for i in chosen_size:
```

Comment:
Are these comments needed?

Reply:
Specifically, the comments from lines 26-28.

This looks like old code that you are just not using anymore. We should probably not include this in the pull request.

Reply:
Thanks Christian. I have removed the comments already!